### PR TITLE
Makes nagibabel prettier :sparkles:

### PR DESCRIPTION
**Pull request by lamartire**
I just added pseudo steps to process, it seems more funny :joy:

### CODE REVIEW

**Comment by lamartire** [comment]
@fual hey, what about merge?)

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+yarn.lock
```
Comment (Dekkee) [inline .gitignore]: yarn.lock must be in the repo

Comment (lamartire) [reply .gitignore]: Solved :raised_hands:

Comment (fual) [reply .gitignore]: > @fual hey, what about merge?)

thank you! That's a best pull request!
But please write the tests and resolve conficlts, please

**Comment by lamartire** [comment]
@fual all done.

**Comment by fual** [comment]
still conflicts, please fix

**Comment by lamartire** [comment]
Reopened as #42

**Pull request closed without merging**